src/projects/EnsureThat/NotNullAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// // --------------------------------------------------------------------------------------------------------------------
```

Comment:
Could we add this either in this file (and rename it so that it can hold many) or as a peer: https://github.com/danielwertheim/Ensure.That/blob/master/src/projects/EnsureThat/Annotations/ValidatedNotNullAttribute.cs